target/linux/sunxi/config-5.4

```diff
@@ -0,0 +1,605 @@
+# CONFIG_AHCI_SUNXI is not set
```

Comment:
Nitpick, it's obvious, but you're saving the commiter some time if you add missing commit description, otherwise the commiter is going to add one anyway.